src/fr/charly/javaTutorial/JavaLesson9.java

```diff
@@ -0,0 +1,21 @@
+package fr.charly.javaTutorial;
+
+public class JavaLesson9 {
+
+	public static void main(String[] args) {
+
+		Apple a = new Apple();
```

Comment:
Avoid variable names as a, b, c,...
A variable name must help the other developers to understand what we're talking about.
So this could be `Apple apple = new Apple();`